### PR TITLE
Bug 1689101: Bump kubernetes-container-terminal to v3.0.1 to pick up …

### PR DESCRIPTION
**Pull request by rhamilto**
…bug fixes

Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1689101

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     "selectize": "0.12.1",
     "c3": "0.4.11",
     "kubernetes-label-selector": "2.1.1",
-    "kubernetes-container-terminal": "2.1.1",
+    "kubernetes-container-terminal": "3.0.1",
```
Comment (spadgett) [inline bower.json:31]: Were there any other changes needed to go from xterm 2.x to xterm 3.x?

Comment (rhamilto) [reply bower.json:31]: Good catch.  Yup.  https://github.com/openshift/origin-web-console/commit/aff114e73d86c02a3db09e3bfc49cd5c2d140498  Lemme update.

Comment (rhamilto) [reply bower.json:31]: Updated.  Thanks, @spadgett.

**Review by spadgett — APPROVED** [review]
👍 thanks @rhamilto 

/lgtm

**Pull request merged**